exercises/tournament/src/lib.rs

```diff
@@ -1 +1,6 @@
-
+pub fn tally(matches_results: &str) -> String {
```

Comment:
Quibble: the argument should be `match_results`; you only need to pluralize the second word, for reasons of English grammar which are out of scope here.